ArchiSteamFarm/WebBrowser.cs

```diff
@@ -343,7 +371,16 @@ internal sealed class WebBrowser : IDisposable {
 			using (HttpRequestMessage request = new HttpRequestMessage(httpMethod, requestUri)) {
 				if (data != null) {
 					try {
-						request.Content = new FormUrlEncodedContent(data);
+						if (!multipartFormData) {
+							request.Content = new FormUrlEncodedContent(data);
```

Comment:
Did you check if your request can't work with `FormUrlEncodedContent`, even if Steam uses multipart? If yes, this is fine, but maybe we can avoid all this mess and just use what we have everywhere else instead.

Reply:
If it requires multipart and can't form with encoded content, then it'd make sense to change type of `data` right to `HttpContent` and do appropriate parsing in functions called earlier.